recipes/geocat-viz/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "2022.05.0" %}
+
+package:
+  name: geocat-viz
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCAR/geocat-viz/archive/v{{ version }}.tar.gz   # For building from a specific tag's tarball
+  sha256: 021e1a7c8a9b6563fff130f58a50e79e8a664bb91d4efb4349811d9ee8f01f40
+  # git_tag: v{{ version }}   # For building from a specific tag
+  # git_rev: main # For building from main branch for debugging purposes
+  # git_url: https://github.com/NCAR/geocat-viz.git
```

Comment:
```suggestion
```
REMINDER: You should always prefer building conda-forge packages from source tarballs.